module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,24 @@
+#include "arithmeticAverage.hpp"
+
+#include <cmath>
+#include <functional>
+#include <numeric>
+
+double ArithmeticAverage(const std::vector<int>& vector1, const std::vector<int>& vector2) {
+    double sum;
+    double result;
+    sum = std::inner_product(vector1.begin(), vector1.end(), vector2.begin(), 0, std::plus<int>(), std::plus<int>());
+    result = sum / (vector1.size() + vector2.size());
+    return result;
```

Comment:
```suggestion
    return std::inner_product(vector1.begin(), vector1.end(), vector2.begin(), 0, std::plus<int>(), std::plus<int>()) / (vector1.size() + vector2.size());
```

Reply:
For me it will be a bit complicated to read, so I will leave it as it is.